packages/waku/tests/tsconfig.json

```
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "../",
+    "noEmit": true
+  },
+  "include": ["../src", "."]
```

can we add another reference here?

---

is this what you mean? 

```suggestion
  "include": ["."],
  "references": [
      {
        "path": "../../../packages/waku"
      },
    ]
``

or do you mean to add a reference to the new tsconfig in the root level tsconfig.json ?

I'm not very familiar with the "references" field just yet honestly

---

I think it will work

---

oh yep! replaced the path with just `".."` but that allows me to remove src from the include. Thanks for the tip!